libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/reader/RtpMp4aReader.java

```diff
@@ -134,7 +140,7 @@ private static long toSampleUs(
    * @throws ParserException If the MPEG-4 Audio Stream Mux configuration cannot be parsed due to
    *     unsupported audioMuxVersion.
    */
-  public static Integer getNumOfSubframesFromMpeg4AudioConfig(
+  private static Integer getNumOfSubframesFromMpeg4AudioConfig(
```

Comment:
You mentioned the streams you tested have got only 0 sub frames, does it mean that the parsing code in this method is never executed?

Reply:
After parsing we are getting number of subframes are zero.